src/chapters/configuring.xml

```diff
@@ -6195,6 +6195,33 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
                 </section>
             </section>
         </section>
+        <section xml:id="parameter-prompting">
+            <title>Parameter prompting</title>
+            <para><indexterm>
+                <primary>parameters</primary>
+                <secondary>prompt</secondary>
+            </indexterm>In certain situations Guacamole may determine that additional
+                information is required in order to successfully open or continue a
+                connection. In these scenarios guacd will send an instruction back to
+                the client to retrieve that information, which will result in the user
+                being prompted for those additional parameters.</para>
+            <para>Currently the only parameters that will trigger this prompt to the
+                user are authentication requests for the RDP and VNC protocols where
+                authenticators were not provided as part of the connection configuration.
+            </para>
+            <important>
+                <para>It is important to note that requests for parameters will only be
+                    generated in the case where that information has not already been
+                    provided as part of the connection. The user will never be asked for
+                    parameters that replace or override connection parameters where
+                    values have been configured as part of the connection, including
+                    authentication information. For example, if a connection to a
+                    RDP server specifies a username and password, and that username
+                    or password is incorrect and results in an  authentication failure,
+                    the connection will fail and the user will not be prompted for
+                    alternative credentials.</para>
```

Comment:
> ... the user will not be prompted for alternative credentials.

Some RDP servers will still take it upon themselves to display a login screen and prompt for correct credentials (if NLA is not in use). It's good to assure the admin that Guacamole will not prompt the user for these things, but I think we should be more specific with that assurance.